spec/api-app-spec.js

```diff
@@ -157,6 +157,24 @@ describe('app module', () => {
         done()
       })
     })
+
+    it('exits gracefully on macos', function (done) {
+      if (process.platform !== 'darwin') {
+        this.skip()
```

Comment:
In general we don't call `skip` on platform specific tests, we just `return` and let the test "pass"

Reply:
@MarshallOfSound Actually we do the opposite )
If a test isn't run, it should we marked as `skipped`, not `passed`.
In rare cases, when we have issues with Mocha, we do call a `return` instead of `this.skip()`.